src/proxy-ui-api/src/main/java/org/niis/xroad/securityserver/restapi/converter/NodeTypeMapping.java

```diff
@@ -0,0 +1,90 @@
+/**
+ * The MIT License
+ * Copyright (c) 2019- Nordic Institute for Interoperability Solutions (NIIS)
+ * Copyright (c) 2018 Estonian Information System Authority (RIA),
+ * Nordic Institute for Interoperability Solutions (NIIS), Population Register Centre (VRK)
+ * Copyright (c) 2015-2017 Estonian Information System Authority (RIA), Population Register Centre (VRK)
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+package org.niis.xroad.securityserver.restapi.converter;
+
+import ee.ria.xroad.common.SystemProperties;
+
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+import org.niis.xroad.securityserver.restapi.openapi.model.NodeType;
+
+import java.util.Arrays;
+import java.util.Optional;
+
+/**
+ * Mapping between NodeType in api (enum) and model (SystemProperties.NodeType)
+ */
+@Getter
+@RequiredArgsConstructor
+public enum NodeTypeMapping {
+    PRIMARY(SystemProperties.NodeType.MASTER, NodeType.PRIMARY),
+    REPLICA(SystemProperties.NodeType.SLAVE, NodeType.REPLICA),
+    STANDALONE(SystemProperties.NodeType.STANDALONE, NodeType.STANDALONE);
```

Comment:
What is the reason/idea behind this term translation?  ie. MASTER -> PRIMARY  , SLAVE -> REPLICA

Reply:
I think we are mostly referring to the master node as `primary` and the slave node as `replica` in the documentation. Thought that might be clearer to the end user.